### ISSUE TITLE
Design mockup of add user to channel

### ISSUE THREAD

**Issue by kevimuri24** [zurichat/zc_plugin_channels#310]

**Comment by oyinolunaike** [comment]
Figma file: https://www.figma.com/file/s5q00IxXZDc9wfOdZ3Un5B/Channels-Plugin-Add-Members?node-id=737%3A8955

Slack username: oyinolunaike
